specs/driver_spec.rb

```diff
@@ -77,4 +77,57 @@
       driver.average_rating.must_equal 0
     end
   end
+
+
+
+
+
+  describe "total_revenue method" do
+    before do
+      @driver = RideShare::Driver.new(id: 3, name: "Lovelace", vin: "12345678912345678")
+      trip = RideShare::Trip.new({id: 8, driver: @driver, passenger: nil, start_time: "2016-08-08", end_time: "2016-08-08T12:16:00+00:00", cost: 17.56, rating: 5})
```

Comment:
Other questions that I would like to see these tests ask, for all the wave 1 methods (on both `Driver` and `Passenger`):

- What happens if the driver or passenger has no trips?
- What happens if the driver or passenger has an incomplete trip?